contentcuration/contentcuration/models.py

```diff
@@ -306,14 +306,12 @@ def get_user_active_trees(self):
 
     def get_user_active_files(self):
         active_trees = self.get_user_active_trees()
-        return self.files.select_related('contentnode')\
-            .filter(Q(contentnode__tree_id__in=active_trees))\
-            .values('checksum', 'file_size')\
-            .distinct()
+        return self.files.filter(contentnode__tree_id__in=active_trees)\
+            .values('checksum', 'file_size')
```

Comment:
Why are we removing the `distinct()` call from this query? Is there any benefit to this function returning duplicate file entries?

Reply:
It provided a small performance improvement on the query - the distinctness is retained by using the DistinctSum whenever we are summing the file size.

Reply:
Yes, but I think it may be unexpected by calling code that `get_user_active_files` returns duplicates, and I could see calling code being written that assumes duplicates were filtered. Case in point, I don't see an update to `check_channel_space`, which also does a `Sum` on the results and would need to be updated to use `DistinctSum` IIUC.

Reply:
I reverted this change, as I think `DistinctSum` is not actually what we want here. Instead, I'm doing the `values` call just on `checksum` now, so the following distinct will distinct by checksum, then we do a sum over `file_size`.

The issue with the previous `DistinctSum` is that it was on distinct file sizes, so technically, someone could upload files with the same file size but different checksum and have them take up no additional space.